lib/register_transformer_dk/config/adapters.rb

```diff
@@ -7,6 +7,11 @@ module Config
     module Adapters
       KINESIS_ADAPTER = RegisterCommon::Adapters::KinesisAdapter.new(credentials: AWS_CREDENTIALS)
       S3_ADAPTER = RegisterCommon::Adapters::S3Adapter.new(credentials: AWS_CREDENTIALS)
+      REDIS_ADAPTER = RegisterCommon::Adapters::Redis.new(
+        host: ENV.fetch('REDIS_HOST'),
```

Comment:
I'd prefer to move to a connection string similar to `ELASTICSEARCH_URL`—e.g. `REDIS_URL` defaulting to `redis://redis:6379` (I'm not sure why `REDIS_HOST` currently is `127.0.0.1` rather than `redis` after the Docker updates…)—but I understand if you consider this out-of-scope for this PR.